### DISCUSSION TITLE
TX Support ?

### DISCUSSION THREAD

**Discussion by Jempiken** [AlexandreRouma/SDRPlusPlus#219] [Ideas]
Hello, Alexandre..  Top Program  , Ergonomic ! Simplicityy ?
Any Ideas for TX support for certain possible SDR's  HackRF and Adalm-Pluto ?
Would be very nice !

JP, ON7MA, Belgium

**Comment by AlexandreRouma** [comment]
Hello,

Unfortunately TX would require a significant architecture change so it's not planned at the moment.

Regards,
Alex

**Reply by Jempiken** [reply]
Tnx Alex for considering.. I understand...
JP
